.github/workflows/ci.yml

```diff
@@ -3,6 +3,10 @@ name: Run CI tests
 on:
   push:
     branches: [dev]
+    paths:
+      - contracts/**
+      - .solcover.js
+      - test/**
```

Comment:
I wonder if it's worth including `cli/` here? Simply because some of the files in `test/` depend on some of the files in `cli/`, so there's a small but nonzero possibility that a cli change will break tests...

(A part of me wants to keep running this particular workflow unconditionally, just to have a single check that we can always require before merging)

Reply:
> (A part of me wants to keep running this particular workflow unconditionally, just to have a single check that we can always require before merging)

You know I was on the fence about this too, let's trust this hunch then.